requirements/anomaly.txt

```diff
@@ -1,4 +1,6 @@
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 # Anomaly Requirements.
+torchvision==0.14.1
+torchtext==0.14.1
```

Comment:
well, now that you just modified this file, maybe you could also replace `anomalib==0.4.0rc2` with `anomalib==0.4.0` too? 😁

Reply:
Reflected! :)